.github/workflows/build.yml

```diff
@@ -275,6 +275,65 @@ jobs:
         run: |
           echo "container_image_uri=${{ fromJSON(steps.meta.outputs.json).tags[0] }}" >> $GITHUB_OUTPUT
 
+  publish:
+    name: Publish
+    needs: [ build, docker ]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          repository: ${{ github.repository }}
+
+      - name: Lowercase environment
+        id: lowercase
+        run: echo TARGET_ENV_LOWERCASE=${{ needs.build.outputs.deploy_env }} | tr '[:upper:]' '[:lower:]' >> "$GITHUB_OUTPUT"
+
+      - name: Publish UMM-S with new version
+        id: publish-umm-s
+        uses: podaac/cmr-umm-updater@0.6.0
+        if: |
+          github.ref == 'refs/heads/main'    ||
```

Comment:
l2ss-py just runs on main and release branches:
https://github.com/podaac/l2ss-py/blob/develop/.github/workflows/build-pipeline.yml#L170-L172

Reply:
I modified the Hydrocron GitHub workflow so that it does not try to publish a record for the SIT environment.